### PR TITLE
feb dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-february-2024.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
+Title: iRODS Development Update: February 2024
+Date: 2024-02-26 12:00
```
Comment (trel) [inline content/posts/irods-development-update-february-2024.md:2]: let's make it 1400 and we'll get it live.

File: content/posts/irods-development-update-february-2024.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-february-2024
+Status: published
+
+
+Hello reader,
```
Comment (trel) [inline content/posts/irods-development-update-february-2024.md:8]: this feels new...

File: content/posts/irods-development-update-february-2024.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+The Metadata Guard rule engine plugin has been updated to fully support the atomic metadata operations API. Work to disallow use of `imeta rmw` and the APIs underlying it is now complete. Users can expect a new release within the coming weeks.
+
+Python iRODS Client 2.0.0 was released. This release brings **pam_password** compatibility with iRODS 4.3, better TTL behavior, the ability to manipulate and query quotas, and more.
+
+iRODS HTTP API 0.2.0 is now available. This release is made up of 57 closed issues based on community feedback and other areas that needed attention. You can view the list of closed issues, see [https://github.com/irods/irods_client_http_api/milestone/2?closed=1](https://github.com/irods/irods_client_http_api/milestone/2?closed=1). As for new development work, we've started working to allow the server to operate as an OAuth Protected Resource. This feature will allow the HTTP API to accept OAuth tokens and verify them, resulting in more flexibility for client applications and a simpler OIDC implementation within the HTTP API.
```
Comment (trel) [inline content/posts/irods-development-update-february-2024.md:16]: "is made up of" -> "includes"

Comment (trel) [inline content/posts/irods-development-update-february-2024.md:16]: "accept OAuth tokens" -> "accept OAuth access tokens"

File: content/posts/irods-development-update-february-2024.md [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+
+For the iRODS S3 API, we've started refactoring it to not rely on C++20 coroutines. There are two reasons for doing this. The first being the development team's lack of knowledge and experience with C++20 coroutines. The other reason being it helps with our transition from libc++ to libstdc++. Aside from that, the S3 API now supports the **DeleteObjects** API endpoint. The last big item for this project is we've started working to add support for Multipart uploads.
```
Comment (trel) [inline content/posts/irods-development-update-february-2024.md:18]: "with C++20 coroutines" -> "with the relatively new C++20 coroutines"

**Review by trel — COMMENTED** [review]
looking real nice.

File: content/posts/irods-development-update-february-2024.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2024.md:8]: Happy to change it.

File: content/posts/irods-development-update-february-2024.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2024.md]: Bumped to 16:00

File: content/posts/irods-development-update-february-2024.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2024.md]: Done

Comment (korydraughn) [reply content/posts/irods-development-update-february-2024.md]: Done

File: content/posts/irods-development-update-february-2024.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2024.md:8]: Is `Hello everyone,` more appropriate/preferred?

Comment (trel) [reply content/posts/irods-development-update-february-2024.md:8]: it's perfectly fine - just made me smile.

**Comment by trel** [comment]
squash em

**Pull request merged**